lua/telescope/_extensions/aerial.lua

```diff
@@ -40,11 +40,12 @@ local function aerial_picker(opts)
     end
   end
 
-  if show_nesting == nil then
-    show_nesting = ext_config.show_nesting["_"]
-  end
-  local backend = backends.get()
+  local show_nesting = (
+    ext_config.show_nesting[filetype] or ext_config.show_nesting["_"]
+  )
```

Comment:
This logic is not the same, as it will fall back to the `_` key if `show_nesting[filetype]` is false, not just if it's `nil`

Reply:
Thanks for the catch, thought I was being smart but turns out maybe not. Reverted the change now.